### PR TITLE
ci: add stale bot

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

# Description

Add stale bot to control issue/pr lifecycle.

It is safe to merge before label, since it would mark those as stale and wait for 37 days to close.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by mukundansundar — APPROVED** [review]
@yaron2 thoughts on this PR?

**Comment by daixiang0** [comment]
ping @yaron2

**Comment by daixiang0** [comment]
@yaron2  please reviw.

**Review by mukundansundar — APPROVED** [review]

File: .github/workflows/dapr-bot-schedule.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# ------------------------------------------------------------
```
Comment (mukundansundar) [inline .github/workflows/dapr-bot-schedule.yml:1]: @daixiang0 Please change the license header ...

Comment (daixiang0) [reply .github/workflows/dapr-bot-schedule.yml]: done.

**Review by mukundansundar — APPROVED** [review]
@yaron2 Can you review this?

**Comment by mukundansundar** [comment]
> @yaron2 Can you review this?

@yaron2 ?

**Review by yaron2 — APPROVED** [review]

**Comment by yaron2** [comment]
Merged, thanks @daixiang0

**Pull request merged**